cmk/notification_plugins/signl4.py

```diff
@@ -24,24 +26,46 @@ def _signl4_url() -> str:
 
 
 def _signl4_msg(context: dict[str, str]) -> dict[str, object]:
-    host_name = context["HOSTNAME"]
-    notification_type = context["NOTIFICATIONTYPE"]
```

Comment:
should be always present, no need for .get()

Reply:
OK, thanks.